.github/workflows/neo4j-to-neptune-ci.yml

```
@@ -0,0 +1,42 @@
+name: Neo4j-to-Neptune CI
+
+on:
+  workflow_dispatch:
+  pull_request:
+    paths:
+      - 'neo4j-to-neptune/**'
+  push:
+    paths:
+      - 'neo4j-to-neptune/**'
```

Consider limiting the workflow to master branch only for these triggers:

```suggestion
    paths:
      - 'neo4j-to-neptune/**'
    branches:
      - master
  push:
    paths:
      - 'neo4j-to-neptune/**'
    branches:
      - master
```